tests/README.md

```diff
@@ -1,24 +1,100 @@
 # Integration tests
 
-Integration tests require a running Kubevirt cluster.  Once you have a running
-Kubevirt cluster, you can use the `-master` and the `-kubeconfig` flags to
-point the tests to the cluster.
+Kubevirt integration tests are a suite of end-to-end tests which run
+against an active Kubevirt cluster.
 
-## Run them on an arbitrary KubeVirt installation
+The tests may execute against various clusters, from integrated kubevirtci
+deployments and up to external kubernetes or OpenShift ones.
+
+## Kubevirtci
+
+Kubevirtci provides containerized Kubernetes clusters which can be used
+to test KubeVirt and similar projects.
 
+KubeVirt integration tests are being run against kubevirtci clusters on CI
+and is the simplest way to run the tests locally in a development environment.
+
+To run the test locally:
+- Configure the desired cluster setup (provider, number of nodes and nics).
+  Note: Some tests may run on on specific configuration and HW.
 ```
-cd tests # from the git repo root folder
-go test -kubeconfig=path/to/my/config -config=default-config.json
+export KUBEVIRT_PROVIDER=k8s-1.17
```

Comment:
We want the user to use the latest provider. I suggest to make this more general, by adding a link to the [k8s provider instruction on kci](https://github.com/kubevirt/kubevirtci/tree/master/cluster-up/cluster/k8s-1.17), we should avoid duplicate instructions that can grow incompatible